sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1573,8 +1569,7 @@ class HistGradientBoostingClassifier(ClassifierMixin, BaseHistGradientBoosting):
 
     Parameters
     ----------
-    loss : {'log_loss', 'auto', 'binary_crossentropy', 'categorical_crossentropy'}, \
-            default='log_loss'
+    loss : {'log_loss'}, default='log_loss'
```

Comment:
API wise, it's a little weird having one option for a parameter. Internally, we do allow for a `BaseLoss` instance, so it's okay.